config/accounts/documents/terms-of-use.md

```diff
@@ -1,87 +1,92 @@
-GOV.UK Platform as a Service (PaaS)
-===================================
+# Terms of Use for GOV.UK Platform as a Service
 
-Terms of Use
-------------
+You will need to read and agree to these Terms of Use before using GOV.UK PaaS.
 
-These terms apply to your and your service's use of GOV.UK PaaS.
+## Changes we've made to the Terms of Use
 
-Before you host a live service or private data on GOV.UK PaaS, your government organisation must have accepted and signed the GOV.UK PaaS data sharing and financial agreement (Memorandum of Understanding).
+**13 January 2017**
+Document approved
```

Comment:
I'm not seeing line breaks when this is deployed on my pipeline.

Reply:
Noticed that yesterday :) Thanks!